Cargo.lock

```diff
@@ -20015,7 +20015,7 @@ dependencies = [
  "sp-version",
  "substrate-test-runtime-client",
  "substrate-wasm-builder",
- "trie-db",
+ "trie-db 0.29.0",
```

Comment:
should have only trie-db here (here both 0.28 and 0.29 are living together in the repo).

Reply:
Also solved here: [build: ⬆️ Upgrade trie-bench from 0.38.0 to 0.39.0](https://github.com/paritytech/polkadot-sdk/pull/3982/commits/6d0016afeadb642435754fd3ebc608e36ae454d3)